src/github/graphql.ts

```diff
@@ -680,7 +703,7 @@ export interface GetChecksResponse {
 				nodes: {
 					commit: {
 						statusCheckRollup?: {
-							state: string;
+							state: 'EXPECTED' | 'ERROR' | 'FAILURE' | 'PENDING' | 'SUCCESS';
```

Comment:
Once again, I do think this is probably a good change, but to keep what needs to be tested for it scoped down could you revert this?

Reply:
Yes. Again, just trying to match it up to the current GraphQL schema.